src/composables/sdk.ts

```diff
@@ -0,0 +1,49 @@
+import type {
+  IDefaultComposableOptions,
+  Protocol,
+} from '../types';
+import { useAccounts } from './accounts';
+import { IChainProtocol } from '../types';
```

Comment:
This can be added to already existing types imports at the beginning of the file.

Reply:
Done.